### ISSUE TITLE
Open braces of initializers should be placed directly below the type position

### ISSUE THREAD

**Issue by RalfKoban** [RalfKoban/MiKo-Analyzers#517]
We should report an issue in case an open brace of an initializer is not placed directly below the type position.
Initializers are no blocks and should not be mixed up with those.

Following example should produce a violation as it looks like a block (such as an `if` block):
```c#
public void DoSomething()
{
    var x = new Something
    {
        SomeProperty = someValue,
    };
}
```

Following example should not produce a violation as the open brace is directly placed below the type information.
```c#
public void DoSomething()
{
    var x = new Something
                {
                    SomeProperty = someValue,
                };
}
```
